Build broken due to error on `openssl-kdf v0.4.1`

```
error: unsupported output in build script of `openssl-kdf v0.4.1`: `cargo::rustc-link-lib=crypto`
Found a `cargo::key=value` build directive which is reserved for future use.
Either change the directive to `cargo:key=value` syntax (note the single `:`) or upgrade your version of Rust.
See https://doc.rust-lang.org/cargo/reference/build-scripts.html#outputs-of-the-build-script for more information about build script outputs.
warning: build failed, waiting for other jobs to finish...
Error: Process completed with exit code 101.
```

--- a/client-linuxapp/src/serviceinfo.rs
+++ b/client-linuxapp/src/serviceinfo.rs
@@ -722,17 +722,14 @@ async fn process_serviceinfo_in(si_in: &ServiceInfo, si_out: &mut ServiceInfo) -
                 sshkey_user.as_ref().unwrap()
             ))?;
         }
-        if sshkey_keys.is_some() {
+        if let Some(sshkey_keys) = sshkey_keys {
             log::info!("SSHkey module was active, installing SSH keys");
             create_user(sshkey_user.as_ref().unwrap()).context(format!(
                 "Error creating new user: {}",
                 sshkey_user.as_ref().unwrap()
             ))?;
-            let sshkey_keys_v: Vec<String> = sshkey_keys
-                .unwrap()
-                .split(';')
-                .map(|s| s.to_string())
-                .collect();
+            let sshkey_keys_v: Vec<String> =
+                sshkey_keys.split(';').map(|s| s.to_string()).collect();
             for key in sshkey_keys_v {
                 let key_s: String = key;
                 install_ssh_key(sshkey_user.as_ref().unwrap(), key_s.as_str())

--- a/data-formats/Cargo.toml
+++ b/data-formats/Cargo.toml
@@ -28,14 +28,14 @@ byteorder = "1"
 
 http = "0.2"
 
-openssl-kdf = { version = "0.4.1", features = ["allow_custom"] }
+openssl-kdf = { version = "0.4.2", features = ["allow_custom"] }
 
 [features]
 # Whether to use a non-interoperable KDF.
 use_noninteroperable_kdf = []
 
 [build-dependencies]
-openssl-kdf = { version = "0.4.1", features = ["allow_custom"] }
+openssl-kdf = { version = "0.4.2", features = ["allow_custom"] }
 
 [dev-dependencies]
 maplit = "1.0"

